src/ngraph/ngraph_emitter.cc

```diff
@@ -695,6 +695,35 @@ void Emitter::CreateBinaryOps() {
     return cast_result(CreateAutoBroadcast<ngraph::op::LessEq>(node),
                        getType(node->dtype_));
   };
+  ngraph_op_funcs_["broadcast_to"] =
+      [this](const NodePtr& node) -> NgraphNodePtr {
+    auto input = op_map_[node->inputs_[0]];
+    const auto& input_shape = input->get_shape();
+    ngraph::Shape output_shape(
+        get_default(node, "shape", std::vector<size_t>{}));
```

Comment:
Is it really possible for the "shape" attribute to be undefined when this code runs?